privaterelay/locales/en-us/privaterelay/app.ftl

```diff
@@ -0,0 +1,226 @@
+#   $url (url) - https://addons.mozilla.org/firefox/addon/private-relay/
+#   $attrs (string) - specific attributes added to external links
+faq-question-1-answer-html = While { -brand-name-relay } does not filter for spam, our email partner Amazon SES does block spam and malware. 
+    If { -brand-name-relay } forwards messages you don’t want, you can update your { -brand-name-relay } settings to block messages from the alias forwarding them. <br/><br/> 
+    If you see a broader problem of unwanted email from all of your aliases, please <a href="{ $url }" { $attrs }>report this to us</a> so we can consider adjusting the SES spam thresholds for this service. If you report these as spam, your email provider will see { -brand-name-relay } as the source of spam, not the original sender.
+faq-question-2-question = Why won’t a site accept my { -brand-name-relay } alias?
+
+# Variables:
+#   $url (url) - https://addons.mozilla.org/firefox/addon/private-relay/
+#   $attrs (string) - specific attributes added to external links
+faq-question-2-answer-html = Some sites may not accept an email address that includes a subdomain (ie, the “relay” portion of @relay.firefox.com) and others have stopped accepting all addresses except those from Gmail, Hotmail, or Yahoo accounts. 
+    As { -brand-name-firefox-relay } grows in popularity and issues more aliases, our service might be placed on a blocklist. 
+    If you are not able to use a { -brand-name-relay } alias, <a href="{ $url }" { $attrs }>please let us know</a>.
+faq-question-3-question = Is { -brand-name-relay } available only in the US?
+faq-question-3-answer = The site is currently only available in English, but you can use the service anywhere.
+faq-question-4-question = Can I reply to messages using my { -brand-name-relay } alias?
+
+# Variables:
+#   $url (url) - https://github.com/mozilla/fx-private-relay/issues/99
+#   $attrs (string) - specific attributes added to external links
+faq-question-4-answer-html = { -brand-name-relay } does not yet offer the ability to reply using an alias. 
+    If you try, nothing will happen. We are planning an additional feature to let you <a href="{ $url }" {$attrs}>reply anonymously to senders</a>.
+faq-question-5-question = Can I make up my own { -brand-name-relay } alias using the @relay.firefox.com domain?
+faq-question-5-answer = Not currently, but we are considering new features including letting you create your own alias with a designated domain.
+faq-question-6-question = What happens if { -brand-name-mozilla } shuts down the { -brand-name-firefox-relay } service?
+faq-question-6-answer = We will give you advance notice that you need to change the email address of any accounts that are using { -brand-name-relay } aliases.
+faq-question-7-question = What if an email sent to my alias contains an attachment?
+faq-question-7-answer = We now support attachment forwarding. However, there is a 150KB limit for email forwarding using { -brand-name-relay }. 
```

Comment:
Possibly a Q for @flodolo and @peiying2, but should the `150KB` be a variable? I think I found 4&times; instances in this .ftl file (here, L128, and L154), meaning that if we change that limit to 300KB at some point, we'd invalidate 4 strings per locale. Perhaps possibly using a `{ -email-size-limit }` variable and change it in one place per locale.

Reply:
Definitely not a bad idea.

Reply:
Agreed. Don't want to update the string when the number changes.

Reply:
Awesome. I'll add it.

Reply:
Added a comment for additional context too. 

```
# The file size limit for emails forwarded through the Relay service. KB is an abbreviation for kilobyte.  
-email-size-limit = 150KB
```